sys/arduino/vendor/Makefile

```diff
@@ -0,0 +1,3 @@
+MODULE = arduino_vendor
```

Comment:
I wonder if this could be a package?

License-wise this seems to be all LGPL2. So one could just add the missing Doxygen doc and make it a first class module. (I assume that printing isn't a bleeding edge feature under active development and this is not going to require much maintenance if any.)

Reply:
These files come from https://github.com/arduino/ArduinoCore-API
So should I make a package of it?

Reply:
If that would be feasble, a package would be IMO be the best option

Reply:
Done. This also fixes the pgmspace.h issue as it's included in ArduinoCore-API